app-containers/incus/incus-6.0.0-r1.ebuild

```diff
@@ -176,11 +186,13 @@ src_install() {
 	systemd_newunit "${FILESDIR}"/incus-user-0.4.service incus-user.service
 	systemd_newunit "${FILESDIR}"/incus-user-0.4.socket incus-user.socket
 
-	# Generate and install shell completion files.
-	mkdir -p "${D}"/usr/share/{bash-completion/completions/,fish/vendor_completions.d/,zsh/site-functions/} || die
-	"${bindir}"/incus completion bash > "${D}"/usr/share/bash-completion/completions/incus || die
-	"${bindir}"/incus completion fish > "${D}"/usr/share/fish/vendor_completions.d/incus.fish || die
-	"${bindir}"/incus completion zsh > "${D}"/usr/share/zsh/site-functions/_incus || die
+	if ! tc-is-cross-compiler; then
+		# Generate and install shell completion files.
+		mkdir -p "${D}"/usr/share/{bash-completion/completions/,fish/vendor_completions.d/,zsh/site-functions/} || die
+		"${bindir}"/incus completion bash > "${D}"/usr/share/bash-completion/completions/incus || die
+		"${bindir}"/incus completion fish > "${D}"/usr/share/fish/vendor_completions.d/incus.fish || die
+		"${bindir}"/incus completion zsh > "${D}"/usr/share/zsh/site-functions/_incus || die
+	fi
```

Comment:
You modify the `bindir` already, isn't this useless now? Whether or not crosscompiling it should resolve to by the former function.

Reply:
I'm gonna add an `ewarn` here about shell completion files not being installed. Rest looks good I guess, thanks!